Gems/PhysX/Code/Source/Material/PhysXMaterial.cpp

```diff
@@ -273,6 +311,72 @@ namespace PhysX
         m_density = AZ::GetClamp(density, MaterialConstants::MinDensityLimit, MaterialConstants::MaxDensityLimit);
     }
 
+    bool Material::IsCompliantContactModeEnabled() const
+    {
+#if (PX_PHYSICS_VERSION_MAJOR >= 5)
+        return m_pxMaterial->getFlags() & physx::PxMaterialFlag::eCOMPLIANT_CONTACT;
```

Comment:
uber nit: Prefer explicit boolean check as opposed to implicit conversion 

```
return (m_pxMaterial->getFlags() & physx::PxMaterialFlag::eCOMPLIANT_CONTACT) != 0;
```

Reply:
done